portfolio/src/main/webapp/map.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Map</title>
+    <link rel="stylesheet" href="common-style.css" />
+    <link rel="stylesheet" href="map-style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet" />
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyARIbGzg1ObeVicxm6Txr1mqhUR4w12LQQ"></script>
```

Comment:
Can you use a key-hiding technique to avoid checking this in? I _think_ that if you use [that one guide](https://gist.github.com/derzorngottes/3b57edc1f996dddcab25), you could read the key in your JS and use it to dynamically construct a script tag to insert into the document.